lib/pseudable.rb

```diff
@@ -1,55 +1,18 @@
 # For models which have pseuds (authors)
 module Pseudable
```

Comment:
My inclination is to move it to HasCreatorships, since it's basically an alternate version of `new_bylines=`. (If I do move it there, do you think I should still cite which module the `new_bylines=` function comes from? Since it'd be in the same module then.)

Reply:
If it ends up in the same place as `new_bylines=` you don't have to mention the module.